http_check/datadog_checks/http_check/data/conf.yaml.example

```diff
@@ -297,10 +297,10 @@ instances:
     #
     # kerberos_cache: <KRB5CCNAME>
 
-    ## @param tls_verify - boolean - optional - default: true
+    ## @param tls_verify - boolean - optional - default: false
     ## Instructs the check to validate the TLS certificate of services.
     #
-    # tls_verify: true
+    # tls_verify: false
```

Comment:
The default seems to be `True`:

https://github.com/DataDog/integrations-core/blob/1bf1bcf5f4af4a54f16bf45ad76d55c4f364ce05/datadog_checks_base/datadog_checks/base/utils/http.py#L69

Reply:
https://github.com/DataDog/integrations-core/blob/master/http_check/datadog_checks/http_check/http_check.py#L41